### PR TITLE
systemd: disable tint-by-default in *spawn and in systemd-run

### PR DESCRIPTION
**Pull request by MingcongBai**
Topic Description
-----------------

- systemd: disable tint-by-default in *spawn and in systemd-run
    Let's face it, it's weird and we have much better ways to notify the user
    about a system running in a container. Change the default behavior (tint by
    default) to only tint when the user specifies a --background= parameter.

Package(s) Affected
-------------------

- systemd: 1:256.4-1

Security Update?
----------------

No

Build Order
-----------

```
#buildit systemd
```

Test Build(s) Done
------------------

**Primary Architectures**

- [x] AMD64 `amd64`
- [x] AArch64 `arm64`
- [x] LoongArch 64-bit `loongarch64`

**Secondary Architectures**

- [x] Loongson 3 `loongson3`
- [x] PowerPC 64-bit (Little Endian) `ppc64el`
- [x] RISC-V 64-bit `riscv64`

### CODE REVIEW

**Comment by MingcongBai** [comment]
Background works, but `--background` seems not to function properly (no matter what color was specified, the background was always tinted blue).

图片

File: app-admin/systemd/autobuild/patches/0002-presets-disable-systemd-networkd-systemd-networkd-wa.patch [hunk 1]
```diff
@@ -23,7 +23,11 @@ index da6b9805fd..dcbcd56a7f 100644
 +disable systemd-networkd-wait-online.service
  enable systemd-nsresourced.socket
  enable systemd-pstore.service
- enable systemd-resolved.service
+-enable systemd-resolved.service
++disable systemd-resolved.service
```
Comment (KexyBiscuit) [inline app-admin/systemd/autobuild/patches/0002-presets-disable-systemd-networkd-systemd-networkd-wa.patch]: Why is this patch still around?

**Review by KexyBiscuit — CHANGES_REQUESTED** [review]

File: app-admin/systemd/autobuild/patches/0002-presets-disable-systemd-networkd-systemd-networkd-wa.patch (hunk 1 above)
Comment (KexyBiscuit) [reply app-admin/systemd/autobuild/patches/0002-presets-disable-systemd-networkd-systemd-networkd-wa.patch]: @OriginCode No consensus was made around this, please revert this patch. Also, move **systemd-networkd** presets to **aosc-os-presets-base** package.

Comment (KexyBiscuit) [reply app-admin/systemd/autobuild/patches/0002-presets-disable-systemd-networkd-systemd-networkd-wa.patch]: @MingcongBai patches have been updated, please rebase this.

**Review by KexyBiscuit — APPROVED** [review]

**Comment by OriginCode** [comment]
LGTM on amd64:
圖片

**Pull request merged**